<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="valueLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enqueueButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enqueue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dequeueButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dequeue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findAndMoveMinButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Find and move MIN</string>
    </property>
   </widget>
   <widget class="QListWidget" name="queueListWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fillRandomButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Random fill</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
